src/osd/PrimaryLogPG.cc

```diff
@@ -5007,10 +5005,19 @@ int PrimaryLogPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
 
   dout(10) << "do_osd_op " << soid << " " << ops << dendl;
 
+  ctx->current_osd_subop_num = 0;
```

Comment:
@dillaman it looks like this line is causing a regression in ceph_test_cls_log:
```
[ RUN      ] cls_rgw.test_log_add_same_time
/home/cbodley/ceph/src/test/cls_log/test_cls_log.cc:144: Failure
      Expected: 10
To be equal to: (int)entries.size()
      Which is: 1
[  FAILED  ] cls_rgw.test_log_add_same_time (1180 ms)
```
could you please help review https://github.com/ceph/ceph/pull/18610, which removes this? that fixes the test, but i'm not sure whether this line belongs somewhere else instead